### PR TITLE
HV-1124 Add integration test for javax.money validation on WildFly

### PR DESCRIPTION
**Pull request by gsmet**
* https://hibernate.atlassian.net/browse/HV-1124

### CODE REVIEW

File: modules/src/script/setupModules.groovy [hunk 1]
```diff
@@ -24,6 +28,7 @@ println "[INFO] Using HV version " + hvArtifactName;
 processFileInplace( hvModuleXml ) { text ->
     text.replaceAll( /hibernate-validator.*jar/, hvArtifactName )
 }
+appendDependency( hvModuleXml, "javax.money.api", true )
```
Comment (gunnarmorling) [inline modules/src/script/setupModules.groovy]: I may misremember, but didn't we want to add this optional dependency to the descriptor to begin with? Then a user would only have to drop in the modules (and add the API module in jboss-deployment-structure.xml), but not fiddle with any existing module.xml.

Comment (gsmet) [inline modules/src/script/setupModules.groovy]: Isn't it what it does? This optional dependency is included in our patch file.

Comment (gunnarmorling) [inline modules/src/script/setupModules.groovy]: Ah, of course. Not sure why, but somehow I thought that'd be the setup code within the integration test module.

Comment (gsmet) [inline modules/src/script/setupModules.groovy]: :)

**Pull request merged**